<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>374</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Обзор</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabLayout">
       <item>
        <layout class="QVBoxLayout" name="dashboardLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Пациенты</string>
      </attribute>
      <layout class="QVBoxLayout" name="patientsLayout">
       <item>
        <layout class="QHBoxLayout" name="searchLayout">
         <item>
          <widget class="QLineEdit" name="searchNameEdit">
           <property name="placeholderText">
            <string>ФИО</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchIdEdit">
           <property name="placeholderText">
            <string>ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="searchDiagEdit">
           <property name="placeholderText">
            <string>Диагноз</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="filterLayout">
         <item>
          <widget class="QComboBox" name="deptBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="statusBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="ageBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="patientsTable"/>
       </item>
       <item>
        <widget class="QGroupBox" name="quickBox">
         <property name="title">
          <string>Быстрые действия</string>
         </property>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QPushButton" name="qaNewTreatmentBtn">
            <property name="text">
             <string>Новое назначение</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="qaVisitBtn">
            <property name="text">
             <string>Записать на прием</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="qaResearchBtn">
            <property name="text">
             <string>Добавить исследование</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="qaEpicrisisBtn">
            <property name="text">
             <string>Создать эпикриз</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_history">
      <attribute name="title">
       <string>История болезней</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTableWidget" name="historyTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_treatments">
      <attribute name="title">
       <string>Назначения</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTableWidget" name="treatmentTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_research">
      <attribute name="title">
       <string>Исследования</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTableWidget" name="researchTable"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
